app/qml/components/MMListView.qml

```diff
@@ -8,13 +8,35 @@
  ***************************************************************************/
 
 import QtQuick
+import QtQuick.Controls
 
 //
 // Hot-fix for hotfix https://github.com/MerginMaps/mobile/issues/3417
 // Seems like there is some issue with cache in ListView
 //
 
 ListView {
+  id: root
 
   cacheBuffer: 0
+  readonly property bool isMobile: (Qt.platform.os === "android" || Qt.platform.os === "ios")
+  property int scrollSpace: !isMobile ? 10 : 0
+
+  ScrollBar.vertical: ScrollBar {
+    id: verticalScrollBar
+
+    policy: ScrollBar.AlwaysOn
+    visible: !isMobile && (root.contentHeight > root.height)
```

Comment:
I'd say it would be better to set the policy to `ScrollBar.AlwaysOff` for mobiles instead of creating it, but hiding later..

```suggestion
    policy: isMobile ? ScrollBar.AlwaysOff : ScrollBar.AlwaysOn
```